bundles/action/org.openhab.action.pushbullet/build.properties

```diff
@@ -0,0 +1,7 @@
+source.. = src/main/java/,\
+           src/main/resources/
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               about.html
```

Comment:
@9037568 It is especially helpful in the binary as this is what is distributed :-)